docs/eventing/broker/alternate/README.md

```diff
@@ -0,0 +1,8 @@
+In the Knative Eventing ecosystem, vendors are welcome to bring their own implementation of Broker
```

Comment:
While I totes agree that vendors are welcome, so are other communities not strictly restricted to vendors. I'd prefer to rework the wording to be more inclusive.

Reply:
Good point! Reworded a little bit.